company-profiles/pexip.md

```diff
@@ -0,0 +1,38 @@
+# Pexip
+
+## Company blurb
+
+Pexip is a video conferencing company focused on security, privacy, and adaptability. We offer both self-hosted and as-a-service deployment options for enterprise video conferencing. We're a company with an engineering centric culture where quality matters more than quantity.
+
+## Company size
+
+600+ (As of March 2022)
+
+## Remote status
+
+As a video conferencing company, enabling people to work from anywhere is what we do.
+With offices around the world, we use our own product every day to stay connected. That being said, we highly encourage socialising and give employees the opportunity to travel to the main offices when they want to be closer to their colleagues.
+
+## Region
+
+Denmark, Finland, Ireland, Switzerland, Poland, Italy, Sweden, Spain, Czech Republic, France, Netherlands, Germany, Belgium and the United Kingdom
```

Comment:
Are these countries named as these are where the company can hire in, or where current employees are located?

Reply:
These are the countries where we can hire from. We have employees around the world but it was meant as the places for remote.